### ISSUE TITLE
[Request] Provide alternative triggering hotkey

### ISSUE THREAD

**Issue by jack980517** [toehead2001/aeroshot#28]
Pressing Alt activates the "keyboard navigation mode" in apps, which causes the dotted outline around controls to be displayed. Window Clippings uses Print Screen alone which doesn't have this problem.

![No outline](https://user-images.githubusercontent.com/4785019/191566495-fb3744c8-bd2b-4204-bd7c-47257944a925.png)

![Outline](https://user-images.githubusercontent.com/4785019/191566521-58788202-b0d6-43d6-b3bf-fdd7284fedfc.png)
